client/src/Movies/SavedList.js

```diff
@@ -10,9 +11,9 @@ export default class SavedList extends Component {
       <div className="saved-list">
         <h3>Saved Movies:</h3>
         {this.props.list.map(movie => (
-          <span className="saved-movie">{movie.title}</span>
+          <Link key={movie.id} to={`/movies/${movie.id}`}><span className="saved-movie">{movie.title}</span></Link>
```

Comment:
What's preferred for readability is breaking components/elements onto separate lines:
```javascript
<Link key={movie.id} to={`/movies/${movie.id}`}>
  <span className="saved-movie">{movie.title}</span>
</Link>
```

Reply:
I also see a great opportunity for destructuring here!
```javascript
{this.props.list.map(({ id, title }) => (
  <Link key={id} to={`/movies/${id}`}>
    <span className="saved-movie">{title}</span>
  </Link>
))}
```